libs/frontend/src/routes/api/supported-languages/+server.ts

```diff
@@ -0,0 +1,11 @@
+import { backendUrls, httpRequestMethod } from "types";
+import { buildBackendUrl } from "@/config/backend";
+import type { RequestEvent } from "./$types";
+
+export async function GET({ fetch }: RequestEvent) {
+	const something = fetch(buildBackendUrl(backendUrls.PUZZLE_LANGUAGES), {
+		method: httpRequestMethod.GET
+	});
+	console.log("hi", { something });
+	return something;
```

Comment:
maybe a better name than something? and remove `console.log()` 

```suggestion
	const supportedPuzzleLanguages = fetch(buildBackendUrl(backendUrls.PUZZLE_LANGUAGES), {
		method: httpRequestMethod.GET
	});

	return supportedPuzzleLanguages;
```